frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -1805,4 +1816,28 @@
         
     }
 
+    angular.module('evalai')
+        .directive('flipClock', [function() {
+            return {
+                replace: true,
+                template: '<div></div>',
+                restrict: 'E',
+                link: function(scope, element) {
+                    setTimeout(function() {
+                        if (countDownTimer <= 0 || countDownTimer == 'moreThanOneMonth') {
+                            countDownTimer = 0;
+                            element.css("display", "none");
+                        }else {
+                            angular.element('#challenge-logo').css('height', '130px');
+                        }
+
+                        element.FlipClock(countDownTimer, {
```

Comment:
<img width="841" alt="Screenshot 2019-05-09 at 5 12 45 AM" src="https://user-images.githubusercontent.com/11664259/57445982-3591b080-7219-11e9-9cd8-bec4ea39715c.png">

Please fix this.

Reply:
Due to this, I'm not able to see the clock.

Reply:
@galipremsagar Can you please clean navigator cache

Reply:
@galipremsagar fixed this error, please check